venues/ICLR.cc/2020/Conference/python/buddy_ac.py

```diff
@@ -0,0 +1,136 @@
+            individual_buddy_group = client.post_group(
+                openreview.Group(
+                    id = individual_buddy_group_id,
+                    readers = [
+                        conference_id,
+                        conference.get_program_chairs_id(),
+                        conference.get_area_chairs_id(paper_number),
+                        individual_buddy_group_id],
+                    nonreaders = [conference_id + '/Paper' + paper_number + '/Authors'],
+                    signatories = [individual_buddy_group_id],
+                    signatures = [conference.get_id()],
+                    writers = [conference.get_id()],
+                    members = []
+                )
+            )
+
+        paper_ac_individual_group = map_paper_ac_individual_groups.get(paper_number)
+        if paper_ac_individual_group and (individual_buddy_group_id not in paper_ac_individual_group.readers):
+            paper_ac_individual_group.readers.append(individual_buddy_group_id)
+            client.post_group(paper_ac_individual_group)
+
+        # Add paper's buddy AC as member to paper's AC parent group
+        paper_ac_parent_group = map_paper_ac_parent_groups[paper_number]
+        client.add_members_to_group(paper_ac_parent_group, individual_buddy_group_id)
+
+        # Add paper's buddy AC as non-invitee for paper's meta-review invitation
+        paper_meta_rev_invitation = map_meta_review_invitations.get(paper_number, None)
+        if (paper_meta_rev_invitation):
+            paper_meta_rev_invitation.noninvitees = [individual_buddy_group_id]
+            client.post_invitation(paper_meta_rev_invitation)
+
+        ## Post AC & AC-Buddy only Comment invitation for this paper
+        ac_conversation_invitation = openreview.Invitation(
+            id = conference_id + '/Paper' + paper_number + '/-/Area_Chair_Only_Comment',
+            readers = [
+                conference.get_area_chairs_id(number = paper_number),
+                conference_id
+            ],
+            writers = [conference_id],
+            signatures = [conference_id],
+            invitees = [
+                conference.get_area_chairs_id(number = paper_number)
+            ],
+            reply = {
+                'forum' : paper.forum,
+                'readers': {
+                    'description': 'Select all user groups that should be able to read this comment.',
+                    'values': [
+                        conference.get_area_chairs_id(number = paper_number),
+                        conference.get_program_chairs_id()
+                    ]
+                    },
+                'writers': {
+                    'values-regex': conference_id + '/Paper' + paper_number + '/(Buddy_)*Area_Chair[0-9]+',
+                    'description': 'How your identity will be displayed.'
+                },
+                'signatures': {
+                    'values-regex': conference_id + '/Paper6/(Buddy_)*Area_Chair[0-9]+',
```

Comment:
This has the paper number hard coded

Reply:
That was a miss from my end. It has been corrected now. Sorry.